paasta_tools/chronos_tools.py

```diff
@@ -257,6 +257,13 @@ def get_disabled(self):
     def get_schedule(self):
         return self.config_dict.get('schedule')
 
+    def get_schedule_interval_in_seconds(self):
+        try:
+            _, _, interval = self.get_schedule().split('/')
```

Comment:
I feel like I'm missing some background information.
What does ISO8601-like schedule have to do with crontab?